pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -171,9 +203,13 @@ def determine_ddp_device_ids(self):
             return None
         return [self.root_device.index]
 
+    def on_save(self, checkpoint: dict) -> dict:
+        return checkpoint
+
     def transfer_distrib_spawn_state_on_fit_end(self, results):
         # TODO: is there a better way than accessing callback through model -> trainer -> callback?
```

Comment:
I think we can remove this TODO.

It makes sense to access them through the trainer. After all, it's the trainer the one handling all callbacks